### ISSUE TITLE
[iOS] snapPhoto file does not exist

### ISSUE THREAD

**Issue by thegamingninja** [mtellect/CameraDeepAR#7]
Problem
---

Trying to read the returned path from `snapPhoto` on iOS throws a `FileSystemException`
Example output:
```
FileSystemException: Cannot open file, path = 'file:///var/moible/rest/of/path.png'  (OS Error: No such file or directory, errno = 2)
```

Steps to Reproduce
---
1. Initialize `DeepArCamera` with `cameraDeepArCallback`
2. Call `controller.snapPhoto()` (on iOS)
3. Create file reference from path `var file = File(imagePath)`
4. Try to read file bytes `file.readAsBytes()`

Possible Cause
---
The `data.write` in `didTakeScreenshot` is called after invoking `onSnapPhotoCompleted`. However, I tried calling `data.write` before `onSnapPhotoCompleted`, and it didn't do anything, but I'm not a swift developer and have limited experience with iOS, so I'm not sure if I was doing anything wrong.

**Comment by Techblogogy** [comment]
I had a similar issue on iOS, turns out I've just had to remove "file://" from path begging and everything worked fine when passing this into File(path)

**Comment by thegamingninja** [comment]
Thanks, I'll give that a try!

**Comment by thegamingninja** [comment]
> I had a similar issue on iOS, turns out I've just had to remove "file://" from path begging and everything worked fine when passing this into File(path)

That worked, Thank you!

I think I'll fork the project to project to fix this by default and some of the other small issues.

**Comment by amalnaami** [comment]
Does this package work well on IOS ?

**Comment by thegamingninja** [comment]
> Does this package work well on IOS ?

Yeah, it works pretty well, you just need to have a function to remove the `file://` from the path of the image before reading it in flutter.

**Comment by mtellect** [comment]
Fixed on this version 0.0.2+3
ReOpen if this issue persist
